From 04d9a4cea699d64b464daef7760de594b1c7f9bf Mon Sep 17 00:00:00 2001
From: Justin Bronder <jsbronder@cold-front.org>
Date: Tue, 1 Mar 2022 17:34:34 -0500
Subject: [PATCH] setup.py: use setuptools instead of distutils

With python 3.10, distutils is deprecated in favor of setuptools and will
be removed in python 3.12.

Upstream-Status: Submitted [sent to maintainer at russell-lrparsing@stuart.id.au]
Signed-off-by: Justin Bronder <jsbronder@cold-front.org>
---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index 89ea344..2848b4f 100644
--- a/setup.py
+++ b/setup.py
@@ -3,7 +3,7 @@
 # Copyright (c) 2013,2014,2015,2016,2017,2018,2021 Russell Stuart.
 # Licensed under GPLv2, or any later version.
 #
-from distutils.core import setup
+from setuptools import setup
 import re
 
 def get_long_description():
